versioned/mongodb/src/test/java/com/dremio/nessie/versioned/store/mongodb/SampleEntities.java

```diff
@@ -0,0 +1,208 @@
+        Entity.ofList(Entity.ofString("addition1"), Entity.ofString("addition2"))))));
+
+    final List<Entity> treeList = new ArrayList<>(L1.SIZE);
+    for (int i = 0; i < L1.SIZE; i++) {
+      treeList.add(ID);
+    }
+
+    final Map<String, Entity> attributeMap = new HashMap<>();
+    attributeMap.put("metadata", ID);
+    attributeMap.put("tree", Entity.ofList(treeList));
+    // The ID is generated from the contents of the L1 object. If any of the contents change, the ID will as well.
+    attributeMap.put("id", Entity.ofBinary(
+        new byte[] {110, 40, 7, 86, -103, 23, 6, -34, -102, 95, -33, 77, -87, 31, 23, 110, 67, -46, 56, -45}));
+    attributeMap.put("keys", Entity.ofMap(keyList));
+    attributeMap.put("parents", Entity.ofList(ID2));
+
+    return L1.SCHEMA.mapToItem(attributeMap);
+  }
+
+  /**
+   * Create a Sample L2 entity.
+   * @return sample L2 entity.
+   */
+  public static L2 createL2() {
+    final List<Entity> treeList = new ArrayList<>(L2.SIZE);
+    for (int i = 0; i < L2.SIZE; i++) {
+      treeList.add(ID);
+    }
+
+    final Map<String, Entity> attributeMap = new HashMap<>();
+    attributeMap.put("id", ID);
+    attributeMap.put("tree", Entity.ofList(treeList));
+
+    return L2.SCHEMA.mapToItem(attributeMap);
+  }
+
+  /**
+   * Create a Sample L3 entity.
+   * @return sample L3 entity.
+   */
+  public static L3 createL3() {
+    final int size = 100;
+    final List<Entity> treeList = new ArrayList<>(size);
+    for (int i = 0; i < size; i++) {
+      treeList.add(Entity.ofMap(ImmutableMap.of(
+          "key", Entity.ofList(Entity.ofString("path1"), Entity.ofString("path2"), Entity.ofString(String.valueOf(i))),
+          "id", ID2
+      )));
+    }
+
+    final Map<String, Entity> attributeMap = new HashMap<>();
+    // The ID is generated from the contents of the L3 object. If any of the contents change, the ID will as well.
+    attributeMap.put("id", Entity.ofBinary(
+        new byte[] {-42, 83, 119, 97, -12, 87, -124, 124, 7, -50, 86, 39, -101, -99, -90, 40, 62, -127, 93, 13}));
+    attributeMap.put("tree", Entity.ofList(treeList));
+
+    return L3.SCHEMA.mapToItem(attributeMap);
+  }
+
+  /**
+   * Create a Sample Fragment entity.
+   * @return sample Fragment entity.
+   */
+  public static Fragment createFragment() {
+    final int size = 10;
+    final List<Entity> keyList = new ArrayList<>(size);
+    for (int i = 0; i < size; i++) {
+      keyList.add(Entity.ofList(Entity.ofString("path1"), Entity.ofString("path2"), Entity.ofString(String.valueOf(i))));
+    }
+
+    final Map<String, Entity> attributeMap = new HashMap<>();
+    attributeMap.put("id", ID);
+    attributeMap.put("keys", Entity.ofList(keyList));
+
+    return Fragment.SCHEMA.mapToItem(attributeMap);
+  }
+
+  /**
+   * Create a Sample Branch (InternalRef) entity.
+   * @return sample Branch (InternalRef) entity.
+   */
+  public static InternalRef createBranch() {
```

Comment:
I suggest you pass in a random object for all of these object constructors to test things.